File: BankConsole1/Engines/Calculation.cs
using BankConsole1.Dtos;
using System;
using System.Collections.Generic;
using System.Text;

namespace BankConsole1.Engines
{
    internal class Calculation
    {
        internal void Addition(AccInfo _accInfo, int amount2)
        {
            _accInfo.Balance += amount2;
        }

        internal void Substraction(AccInfo _accInfo, int amount) 
        {
            if (0 < _accInfo.Balance - amount)
            {
                _accInfo.Balance -= amount;
            }
            else { Console.WriteLine("Insuffient Balance"); }
        }
    }
}

File: bank_console_python/engines/calculation.py
from dtos.acc_info import AccInfo


class Calculation:
    def addition(self, acc_info: AccInfo, amount: int):
        acc_info.balance += amount

    def substraction(self, acc_info: AccInfo, amount: int):
        if 0 < acc_info.balance - amount:
            acc_info.balance -= amount
        else:
            print("Insuffient Balance")
